tests/pkg_openwsn_debugpins/main.c

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
I assume some of these tests are taken from OpenWSN? Most are missing RIOTized license headers.

Reply:
Doesn't apply